src/forms/TimeInput.jsx

```diff
@@ -20,56 +20,56 @@ const formatDigits = (number) => `0${number}`.slice(-2);
  * @returns {String|Object} the only parsed value or all the parsed parts if no part defined
  */
 export const getTimeParts = (time) => {
-	const [ hours, minutes ] = time.split(':');
-	return {
-		hours,
-		minutes,
-		meridian: hours < 12 ? 'AM': 'PM',
-	};
+  const [hours, minutes] = time.split(':');
```

Comment:
looks like some bad Prettier reformatting - everything should remain tabs, not spaces - make sure you check your PRs when you first set them up to catch little things like this

Reply:
Fixed it, that was an editor setting. I assumed it had the pre commit hooks to reformat like mup-web, my bad.